instrumentation/clickhouse/clickhouse-client/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/clickhouse/ClickHouseAttributesGetter.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.clickhouse;
+
+import io.opentelemetry.api.internal.StringUtils;
+import io.opentelemetry.instrumentation.api.incubator.semconv.db.DbClientAttributesGetter;
+import io.opentelemetry.semconv.incubating.DbIncubatingAttributes;
+import javax.annotation.Nullable;
+
+final class ClickHouseAttributesGetter implements DbClientAttributesGetter<ClickHouseDbRequest> {
+  @Nullable
+  @Override
+  public String getStatement(ClickHouseDbRequest request) {
+    if (request.getSqlStatementInfo() == null) {
+      return null;
+    }
+    String statement = request.getSqlStatementInfo().getFullStatement();
+    return StringUtils.isNullOrEmpty(statement) ? null : statement;
```

Comment:
elsewhere where `getFullStatement` is used we don't do empty string checks, I think you can remove it from this method and just return the result of `getFullStatement` . The same applies for `getOperation`.